prisma/schema.prisma

```diff
@@ -187,7 +188,9 @@ model Exercise {
   answer           String
   testStr          String?
   explanation      String?
-  removed          Boolean?             @default(false)
+  removedAt        DateTime?
```

Comment:
It's only possible to set `removedAt` default value to a `DateTime`, and in the client, we will use it like the following:

```
if (exercise.removedAt) // do something
```

If it has a default, value, the above condition will be true, and we'd get an opposite behavior for what we want.

Reply:
If you don't set a default value postgresql will set it to `null` so we don't need to explicitly set the default to null in prisma,  postgresql will do it and we'll get the desired behaviour. Being limited to datetime as a default value is due to prisma, prisma doesn't let you set null as the default for an optional value but that is the default behaviour.
Just adding `removedAt DateTime?` like you have done is enough.